src/components/ConversationSettings/ConversationAvatarEditor.vue

```diff
@@ -320,11 +320,11 @@ export default {
 			}
 		},
 
-		savePictureAvatar() {
+		async savePictureAvatar() {
 			this.showCropper = false
 			const canvasData = this.$refs.cropper.getCroppedCanvas()
 			const scaleFactor = canvasData.width > 512 ? 512 / canvasData.width : 1
-			this.$refs.cropper.scale(scaleFactor, scaleFactor).getCroppedCanvas().toBlob(async (blob) => {
+			await this.$refs.cropper.scale(scaleFactor, scaleFactor).getCroppedCanvas().toBlob(async (blob) => {
```

Comment:
`toBlob` does not return any `promise`
https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob